tests/data/submit_tuxbuild/build-x86-gcc/build.json

```diff
@@ -0,0 +1,54 @@
+[
+    {
+        "build_name": "lkft-build-x86-gcc",
```

Comment:
is this a real name ?
Can't we just pick a later version so the name reflects the name we actually have gcc-11-lkftconfig ?

Reply:
Done